Setup for rehype plugins (#940)
SAP/luigi: website/docs/scripts/mocks/plain.md

# Hello Luigi

> A standard file with some links, but not any special features

Reviewer: ```suggestion
`Hello Luigi` is a standard file with some links, but no special features.
```

Choose the framework to build your application:

[Application setup without a framework](#noframework) <br>
[Angular 6](#angular6)<br>

> Follow these steps to build a web application based on Luigi:

Author: Well, this file is anyway just a mock file, not any real type of documentation. Maybe I should just add some "lorem ipsum" there.

> Follow these steps to build a web application based on Luigi:

Reviewer: Oh, ok! No problem then 🙂

> 1. Add Luigi npm packages to your project dependencies. 

Reviewer: ```suggestion
```
